client/my-sites/customer-home/cards/features/help-search/index.jsx

```diff
@@ -40,7 +40,7 @@ const HelpSearch = ( { searchQuery, track } ) => {
 	const translate = useTranslate();
 
 	// trackResultView: Given a result, send an "_open" tracking event indicating that result is opened.
-	const trackResultView = ( event, result ) => {
+	const trackResultView = ( result ) => {
```

Comment:
We're not using this parameter anymore, but the `<InlineHelpSearchResults>` component is still _calling_ this function with both parameters; so we need `event` to stay there so that `result` will be the second parameter. It's too bad this file isn't using TypeScript because it would have caught this. We also can't change `<InlineHelpSearchResults>` to remove the `event` parameter from the function call because another user of the results component uses the event parameter.

You can see the bug this is causing if you filter the network dev tools for `t.gif` and look for the tracks event that's recorded when clicking a support link. The event name should be `calypso_inlinehelp_article_open`, because `result.type === 'article'`. However now it's `calypso_inlinehelp_click_open` because `result.type === 'click'`, since the `result` parameter now represents the DOM click event.

<img width="778" alt="unt" src="https://user-images.githubusercontent.com/1500769/124528277-efb5ee80-de5b-11eb-95ec-9c5aced6cd2f.png">

Reply:
Ah, thanks! I've restored that now.